### ISSUE TITLE
[Android][Accessibility] WCAG 2.5.5: Touch inputs must have a sufficient target size. (android.widget.Spinner)

### ISSUE THREAD

**Issue by golddove** [microsoft/AdaptiveCards#4111]
## Platform
* Android

## Version
master

#### Issue
`Touch inputs must have a sufficient target size.` ([TouchSizeWcag](https://accessibilityinsights.io/info-examples/android/touch-size-wcag/))

![image](https://user-images.githubusercontent.com/4442788/83781364-6a5e7a00-a65c-11ea-918e-3d797c3cf696.png)

#### Target application
io.adaptivecards.adaptivecardssample
#### Element path
android.widget.Spinner
#### How to fix
    The element has an insufficient target size (width: 330.666dp, height: 24dp). Set the element's minWidth and minHeight attributes to at least 44dp.
#### Environment

This accessibility issue was found using Accessibility Insights for Android 2020.527.10 (axe-android), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
